pygit2/repository.py

```diff
@@ -56,8 +56,10 @@
 
 class Repository(_Repository):
 
-    def __init__(self, *args, **kwargs):
-        super(Repository, self).__init__(*args, **kwargs)
+    def __init__(self, path, *args, **kwargs):
+        if not isinstance(path, str):
```

Comment:
Shouldn't it be something like ``six.string_types`` here? Because this will surely not work/break on py3